frontend/src/v5/ui/routes/dashboard/projects/federations/federationsList/federationListItem/federationEllipsisMenu/federationEllipsisMenu.component.tsx

```diff
@@ -0,0 +1,128 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import { useParams } from 'react-router';
+import { formatMessage } from '@/v5/services/intl';
+import { DialogsActions } from '@/v5/store/dialogs/dialogs.redux';
+import { useDispatch } from 'react-redux';
+import { EllipsisMenu } from '@controls/ellipsisMenu/ellipsisMenu.component';
+import { EllipsisMenuItemLink } from '@controls/ellipsisMenu/ellipsisMenuItem/ellipsisMenutItem.component';
+import { IFederation } from '@/v5/store/federations/federations.types';
+import { FederationsActionsDispatchers } from '@/v5/services/actionsDispatchers/federationsActions.dispatchers';
+
+type FederationEllipsisMenuProps = {
+	federation: IFederation,
+};
+
+export const FederationEllipsisMenu = ({
+	federation,
+}: FederationEllipsisMenuProps) => {
+	const { teamspace, project } = useParams() as { teamspace: string, project: string };
+	const dispatch = useDispatch();
+
+	return (
+		<EllipsisMenu>
+			<EllipsisMenuItemLink
+				title={formatMessage({
+					id: 'federations.ellipsisMenu.loadFederation',
+					defaultMessage: 'Load Federation in 3D Viewer',
+				})}
+				to={`/${federation._id}`}
+				onClick={() => { }}
+			/>
+
+			<EllipsisMenuItemLink
+				title={formatMessage({
+					id: 'federations.ellipsisMenu.edit',
+					defaultMessage: 'Edit Federation',
+				})}
+				to=""
+				onClick={() => { }}
```

Comment:
lets get rid of the empty tos and onclick if posible

Reply:
done